examples/s3_access.ipynb

```diff
@@ -81,6 +83,7 @@
    "source": [
     "results = harmony_client.result_urls(job_id, link_type=LinkType.s3)\n",
     "print(results)\n",
+    "# NOTE: if you specified destination_url you'll have to retrieve your credentials in another manner\n",
```

Comment:
What is that other manner? Could we add more explanation in markdown to offer a little more guidance? I would recommend we update intro_tutorial.ipynb as well - that notebook was intended to be the primary tutorial notebook when harmony-py was developed, as it provided more end-user guidance on all of the various parameters and access mechanisms (i.e. meant to be more of the comprehensive end-to-end tutorial, using production examples, etc.).

Reply:
Added some further guidance. https://github.com/nasa/harmony-py/pull/55/commits/daaacd63f28ad7d4acc775c5b6a26d674ed7c056